ui-tests/common/com.liferay.ide.ui.liferay/src/com/liferay/ide/ui/liferay/SwtbotBase.java

```diff
@@ -83,7 +83,7 @@ public static void beforeClass() throws Exception {
 
 		SWTBotPreferences.TIMEOUT = 30 * 1000;
 
-		System.setProperty(SWTBotPreferenceConstants.KEY_TIMEOUT, "30000");
```

Comment:
Why should we change this?

Reply:
When running, the current state timeout is displayed